app/views/DateTimeFormats.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import java.time.format.DateTimeFormatter
+import java.time.{LocalDate, LocalDateTime, ZoneId}
+
+object DateTimeFormats {
+
+  private val dateFormatter               = DateTimeFormatter.ofPattern("d MMMM yyyy")
+  def formatDate(date: LocalDate): String = dateFormatter.format(date)
+
+  private val ukZone: ZoneId      = ZoneId.of("Europe/London")
+  private val dateAtTimeFormatter = DateTimeFormatter.ofPattern("d MMMM uuu 'at' h:mma").withZone(ukZone)
+
+  def formatDateAtTime(datetime: LocalDateTime): String =
```

Comment:
I have had a good look for an automatic way of doing this. Found nothing to top this simple and easy to understand approach for AM am PM pm

Reply:
Yeah, there is no easy way.  It's some code I borrowed from NIDAC (which in turn I borrowed from a previous project) which is required to format date-time according to these GDS recommendations - https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#times